code/modules/mob/living/carbon/life.dm

```diff
@@ -9,7 +9,7 @@
 		. = FALSE
 	if(!.)
 		return
-	handle_blood()
+	handle_blood(seconds, times_fired)
```

Comment:
i know it's out of scope but rename this var to delta_time i beg you (it's not actually necessary though)

Reply:
I would need to touch it elsewhere, as i'm just grabbing a proc argument from the mob subsystem

Reply:
Actually, it should probably be fine yeah, lemme see...